internal/tests/cluster/ipv6_listener_test.go

```diff
@@ -77,16 +77,18 @@ func TestIPv6Listener(t *testing.T) {
 	time.Sleep(10 * time.Second)
 	expectWorkers(c1, w1)
 
-	require.NoError(w1.Worker().Shutdown(true))
-	time.Sleep(10 * time.Second)
-	expectWorkers(c1)
+	c2 := c1.AddClusterControllerMember(t, &controller.TestControllerOpts{
```

Comment:
Tests in this `cluster` folder relied on calling `Shutdown(true)` on a particular controller, doing some assertions and then starting the same controller back up. This is no longer possible due to the changes, so instead we spin up a new controller and perform similar assertions against it.